packages/web-components/src/player-component/player-component.ts

```diff
@@ -483,10 +483,48 @@ export class PlayerComponent extends FASTElement {
         }
     }
 
+    private updateNextDayDisabled(): null {
```

Comment:
>: null [](http://example.com/codeflow?start=35&length=6)

why are you returning type null here?
 remove it so that you don't have to return null

Reply:
This return null is removed.